tests/Core/RandomExtensionTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using FsCheck.Xunit;
+using Xunit;
+
+namespace Bearded.Utilities.Tests.Core
+{
+    public class RandomExtensionTests
+    {
+        [Property]
+        public void NextLong(int seed)
+        {
+            var random = new Random(seed);
+
+            random.NextLong();
+        }
+
+        [Property]
+        public void NextLongWithMax(int seed, long max)
+        {
+            var random = new Random(seed);
+
+            var number = random.NextLong(max);
+            
+            Assert.InRange(number, 0, max - 1);
+        }
+
+        [Property]
+        public void NextLongWithMinAndMax(int seed, long min, long max)
+        {
+            var random = new Random(seed);
+
+            var number = random.NextLong(min, max);
+
+            Assert.InRange(number, min, max - 1);
+        }
+
+        [Property]
+        public void NextDoubleWithMax(int seed, double max)
+        {
+            var random = new Random(seed);
+
+            var number = random.NextDouble(max);
+
+            Assert.InRange(number, 0, max);
+            Assert.NotEqual(max, number);
+        }
+
+        [Property]
+        public void NextDoubleWithMinAndMax(int seed, double min, double max)
+        {
+            var random = new Random(seed);
+
+            var number = random.NextDouble(min, max);
+
+            Assert.InRange(number, 0, max);
+            Assert.NotEqual(max, number);
+        }
+
+        [Property]
+        public void NextFloat(int seed)
+        {
+            var random = new Random(seed);
+
+            random.NextFloat();
```

Comment:
Same for NextLong above I suppose, but I find it a bit odd to have a test without a single assertion. Should we maybe at least check if the result is a valid float? Assert.IsNotNaN or something of the like?